plugin/src/main/java/net/dzikoysk/funnyguilds/guild/Guild.java

```diff
@@ -180,15 +175,19 @@ public void setDeputies(Set<User> users) {
     }
 
     public void setRegion(Region region) {
-        if (! FunnyGuilds.getInstance().getPluginConfiguration().regionsEnabled) {
+        PluginConfiguration config = FunnyGuilds.getInstance().getPluginConfiguration();
+
+        if (! config.regionsEnabled) {
             return;
         }
 
         this.region = region;
         this.region.setGuild(this);
 
+        Location center = region.getCenter();
+
         if (this.home == null) {
-            this.home = region.getCenter();
+            this.home = center.clone();
```

Comment:
To w sumie musi być clonowane?

Reply:
klonuje bo jak ktoś coś zmieni to później będzie trzeba szukać o co chodzi. A te lokacje bukkita to mutable są i już miałem problem z hologramem co mi spieprzał xD